asl.py

```diff
@@ -1,9 +1,11 @@
-from flask import Flask, request
+import tensorflow as tf
+import numpy as np
+from flask import Flask, request, jsonify
 from .models import initial_model
 from .utilities import process_img
 
-model = initial_model()  # created upon deployment instead of every call of predict
 
+model = initial_model()  # created upon deployment instead of every call of predict
```

Comment:
Thank you so much for optimizing this function, as well as commenting about how it's performing.